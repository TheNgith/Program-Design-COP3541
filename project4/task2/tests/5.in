4 6 8 12 4 2 8 14
