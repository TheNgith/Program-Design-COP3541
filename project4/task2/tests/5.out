Enter length of array #1:Enter array elements: Enter length of array #2:Enter array elements: Output: 6 12 4 14